ROADMAP.md

```diff
@@ -0,0 +1,3 @@
+# Roadmap
+
+Project roadmap can be found in the associated [Kyverno project view](https://github.com/kyverno/kyverno/projects/8)
```

Comment:
I am not sure if the project view is maintained. @chipzoller - can you please confirm?

We can state something like this:
> Kyverno features are tracked via (Kyverno Design Proposals [KDP])[https://github.com/kyverno/KDP] and milestones on GitHub. This link provides the currently planned milestones sorted by the closest due date:
> https://github.com/kyverno/kyverno/milestones?direction=asc&sort=due_date&state=open

Reply:
Oh, I got it, applying the changes in a while.